GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+@PART[FGB_SolarPanel]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	
+	@mass = 0.144
+	@maxTemp = 1073.15
+	@MODULE[ModuleDeployableSolarPanel]
+            {
+            	@chargeRate = 1.644
+            }
+}
+
+
+@PART[FGB_Polyus]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+    @CoMOffset = 0.0, 3.55, -0.0
+	@mass = 7.3665
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+
+//	!RESOURCE[ElectricCharge]
+//	{
+//	}		
+	!RESOURCE[MonoPropellant]
+	{
+	}	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	%ullage = True
+	%pressureFed = True
+	%ignitions = 55
+	!IGNITOR_RESOURCE,* {}
+	IGNITOR_RESOURCE
+	{
+	}   //
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 15000.999
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = NTO
+			amount = 9645.175
+			maxAmount = 9645.175
+		}
+		TANK
+		{
+			name = UDMH
+			amount = 5355.824
+			maxAmount = 5355.824
+		}
+
+        TANK
+        {
+            name = ElectricCharge
+            amount = Full
+            maxAmount = 90000
+        }
+       }
+       !MODULE[ModuleRCS]
+       { }
+       MODULE
+       {
+         name = ModuleRCS
+         thrusterTransformName = RCSthruster
+         thrusterPower = 0.72
+         PROPELLANT
+         {
+           name = UDMH
+           ratio = 0.357
+         }
+         PROPELLANT
+         {
+           name = NTO
+           ratio = 0.643
+         }
+         atmosphereCurve
+         {
+           key = 0 253
+           key = 1 100
+         }
+     
+       }
+       @MODULE[ModuleEngines]         //  @MODULE[ModuleEngines*]
+       {
+         @minThrust = 17.52
+         @maxThrust = 17.52
+         @PROPELLANT[MonoPropellant]
+         {
+         @name = UDMH
+         @ratio = 0.357
+         }
+         %PROPELLANT
+         {
+           name = NTO
+           ratio = 0.643
+         }
+         @atmosphereCurve
+         {
+           @key,0 = 0 305
+           @key,1 = 1 112
+         }
+       }
+       MODULE
+       {
+         name = ModuleEngineConfigs
+         type = ModuleEngines
+         configuration = KRD-442 Polyus
+         modded = false
+         CONFIG
+         {
+           name = KRD-442 Polyus
+           minThrust = 0.0
+           maxThrust = 17.52
+           heatProduction = 100
+           PROPELLANT
+           {
+     	name = UDMH
+     	ratio = 0.357
+     	DrawGauge = True
+           }
+           PROPELLANT
+           {
+     	name = NTO
+     	ratio = 0.643
+           }
+           atmosphereCurve
+           {
+     	key = 0 302
+     	key = 1 112
+        }
+      }

```

Comment:
Formatting.  Also filling these values in the MEC means they don't need to be added to the ModuleEngines* -- doing so causes confusion, especially when there is conflicting data [for instance, the minThrust settings]